README.md

```diff
@@ -40,10 +40,15 @@ Feel free to follow the next action points:
 * Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
   Your private key will be generated in `./cert/pki/private`
 
-  > ### This key **must** remain **secret**!
+  > ⚠️ This key **must** remain **secret**! ⚠️
+
+* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
+* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn)
```

Comment:
```suggestion
* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn)
* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN, and referencing your pull request
```

Reply:
Reverted the order so the issue can contains a link to the PR previously created

Reply:
Sorry to disagree, but an issue should be open first, and then amended.

Opening a PR before the associated issue feels really weird 😅

(my rule of thumb is "start with why" and "issue == why + what | PR == how")